back/src/Model/Websocket/ExtRooms.ts

```diff
@@ -40,7 +41,11 @@ let RefreshUserPositionFunction = function(rooms : ExtRooms, Io: socketIO.Server
 
         // update position in the worl
         let messageUserPosition = new MessageUserPosition(data);
-        World.updatePosition(messageUserPosition);
+        let world = Worlds.get(messageUserPosition.roomId);
```

Comment:
We could store the roomId directly in the socket of the user. This way, there is no need to send the roomId on the wire on every move.

Reply:
roomId is already saved in the socket.  I have made this code here, because it's here that you refresh mapping position of all users. And in this code we save for each room each position of users.
Here we are in the Rooms socket what represent all sockets created and activated in the back.

Reply:
Oops sorry, shouldn't be drinking and reviewing PR at the same time :D